common/ast/tests/it/testdata/statement.txt

```diff
@@ -3779,3 +3779,29 @@ ShowGrants {
 }
 
 
+---------- Input ----------
+REVOKE SELECT, CREATE ON * FROM 'test-grant'@'localhost';
+---------- Output ---------
+REVOKE SELECT, CREATE ON * FROM USER 'test-grant'@'localhost'
+---------- AST ------------
```

Comment:
Output is mismatched. Is it intended?

Reply:
#revoke : "`REVOKE { ROLE <role_name> | schemaObjectPrivileges | ALL [ PRIVILEGES ] ON <privileges_level> } FROM { [ROLE <role_name>] | [USER] <user> }`"

Now we can revoke user privs and not use keyword `USER` . 

like this sql

```
grant all on *.* to 'test'; -- actually it will grant all privileges to user 'test'@'%';
revoke all on *.* from 'test' -- actually it will revoke all privileges from user 'test'@'%';
```
But I think if we can output the keyword `USER`.

This makes it more clear what object we're operating on.